themes/classic/templates/catalog/_partials/category-footer.tpl

```diff
@@ -0,0 +1,31 @@
+{**
+ * Copyright since 2007 PrestaShop SA and Contributors
+ * PrestaShop is an International Registered Trademark & Property of PrestaShop SA
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Academic Free License 3.0 (AFL-3.0)
+ * that is bundled with this package in the file LICENSE.md.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/AFL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to https://devdocs.prestashop.com/ for more information.
+ *
+ * @author    PrestaShop SA and Contributors <contact@prestashop.com>
+ * @copyright Since 2007 PrestaShop SA and Contributors
+ * @license   https://opensource.org/licenses/AFL-3.0 Academic Free License 3.0 (AFL-3.0)
+ *}
+{if $category.additional_description}
```

Comment:
@Hlavtox this description shouldn't be displayed on the 2nd page, right?

Reply:
@kpodemski No it should be displayed all the time, the same as the upper content. Why hide it?

Reply:
@Hlavtox to avoid duplicate content, upper description is hidden on the second page by default

https://github.com/PrestaShop/PrestaShop/blob/db334ee2069e946be7703cbca4b1168681d5a86b/themes/classic/templates/catalog/_partials/category-header.tpl#L26